LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2023 Google LLC
```

Comment:
I don't think this is saying who actually has copyright, it's just a template to apply. See the paragraph above.

Reply:
Gotcha, closing this out